src/display.rs

```diff
@@ -162,6 +163,24 @@ impl Display for RawTransaction {
     }
 }
 
+impl Display for NodeInfo {
+    fn to_json(&self) -> Json {
+        let s: Vec<&str> = self
+            .net_info
+            .as_ref()
+            .unwrap()
+            .multi_address
```

Comment:
multiaddr parse use [tentacle-multiadd lib](https://crates.io/crates/tentacle-multiaddr)

Reply:
due to 2021 edtion, use version 0.3.0